File: Exercise/Easy/14. Simple Calculator/Code/Simple Calculator.java
import java.util.Scanner;
import java.lang.Math;

public class SimpleCalculator {

    public static void simpleCalculator(double num1, double num2, char operatorChar) {
        double result;
        switch (operatorChar) {
            case '+':
                result = num1 + num2;
                System.out.println(result);
                break;
            case '-':
                result = num1 - num2;
                System.out.println(result);
                break;
            case '*':
                result = num1 * num2;
                System.out.println(result);
                break;
            case '/':
                if (num2 != 0) {
                    result = num1 / num2;
                    System.out.println(result);
                } else {
                    System.out.println("Cannot divide by zero");
                }
                break;
            case '%':
                if (num2 != 0) {
                    result = num1 % num2;
                    System.out.println(result);
                } else {
                    System.out.println("Cannot divide by zero");
                }
                break;
            case '^':
                result = Math.pow(num1, num2);
                System.out.println(result);
                break;
            default:
                System.out.println("Invalid operator");
                break;
        }
    }

    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        double num1 = scanner.nextDouble();
        double num2 = scanner.nextDouble();
        char operatorChar = scanner.next().charAt(0);

        simpleCalculator(num1, num2, operatorChar);
    }
}

File: Exercise/Easy/14. Simple Calculator/Code/Simple Calculator.py
import math

def simple_calculator(num1, num2, operator_char):
    result = 0
    if operator_char == '+':
        result = num1 + num2
        print(result)
    elif operator_char == '-':
        result = num1 - num2
        print(result)
    elif operator_char == '*':
        result = num1 * num2
        print(result)
    elif operator_char == '/':
        if num2 != 0:
            result = num1 / num2
            print(result)
        else:
            print("Cannot divide by zero")
    elif operator_char == '%':
        if int(num2) != 0:
            result = int(num1) % int(num2)
            print(result)
        else:
            print("Cannot divide by zero")
    elif operator_char == '^':
        result = math.pow(num1, num2)
        print(result)
    else:
        print("Invalid operator")

if __name__ == "__main__":
    num1 = float(input())
    num2 = float(input())
    operator_char = input()

    simple_calculator(num1, num2, operator_char)